src/ui/layout/mct-tree.vue

```diff
@@ -143,15 +143,13 @@ export default {
             ancestors: [],
             childrenSlideClass: 'down',
             availableContainerHeight: 0,
-            noScroll: true,
             updatingView: false,
+            itemHeightCalculated: false,
             itemHeight: 28,
             itemOffset: 0,
-            childrenHeight: 0,
             scrollable: undefined,
-            pageThreshold: 50,
             activeSearch: false,
-            getChildHeight: false,
+            getItemHeight: false,
```

Comment:
we should rename `getItemHeight` to something like `enableGetItemHeight` or `isGetItemHeight` etc as its a boolean.
Also other boolean props. `getItemHeight` sound more like a function returning itemHeight

Reply:
same for `:emit-height` in 'tree-item'

Reply:
Good call! updated to shouldEmitHeight.